OasisDEX/oasis-borrow
# feat: check window width in groove widget

# [Title](https://shortcutLink)

https://app.shortcut.com/oazo-apps/story/12211/remove-the-groove-chat-with-us-widget-from-mobile-size-screens
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
- check if the widget shows up on mobile window size

it rewuires reload on window size change
